### PR TITLE
Mempool api.

### PR DESCRIPTION
**Pull request by jgimeno**

### CODE REVIEW

**Review by alessio — APPROVED** [review]

File: cosmos/launchpad/mempool_test.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package launchpad
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+
```
Comment (ilgooz) [inline cosmos/launchpad/mempool_test.go:10]: Not blocking, we can rm empty import lines and rerun go imports

File: cosmos/launchpad/mempool_test.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	"github.com/tendermint/cosmos-rosetta-gateway/cosmos/launchpad/client/tendermint/mocks"
+)
+
+func TestLaunchpad_Mempool(t *testing.T) {
+	m := &mocks.TendermintInfoAPI{}
+	defer m.AssertExpectations(t)
+
+	m.
+		On("UnconfirmedTxs", mock.Anything, mock.Anything).
```
Comment (ilgooz) [inline cosmos/launchpad/mempool_test.go:24]: Not blocking

```suggestion
		On("UnconfirmedTxs", mock.Anything, nil).
```

Comment (jgimeno) [reply cosmos/launchpad/mempool_test.go:24]: It does not work then.

Comment (ilgooz) [reply cosmos/launchpad/mempool_test.go:24]: it is probably nil has to have type
```
opt := *tendermintclient.UnconfirmedTxsOpts
On("UnconfirmedTxs", mock.Anything, opt).
```

**Comment by jgimeno** [comment]
@ilgooz can we do something about this closes?

**Comment by ilgooz** [comment]
> @ilgooz can we do something about this closes?

@fadeev any ideas?

**Comment by fadeev** [comment]
@ilgooz Asana treats both issues and PRs as tasks to be synced. I've selected only "Issues", let's see if that helps.

**Review by ilgooz — APPROVED** [review]

**Pull request merged**